### DISCUSSION TITLE
Can't assign name to certificate

### DISCUSSION THREAD

**Discussion by poksiala** [Kong/kong#7278] [Ideas and feature requests]
### Summary

According to SNI documentation you can reference certificate by name, but `/certificates` api endpoint does not accept name field to be assigned to certificate. It would be great if we could reference certificates by name

https://docs.konghq.com/1.1.x/admin-api/#create-sni-associated-to-a-specific-certificate

### Steps To Reproduce

1. Try to create certificate with name field

### Additional Details & Logs

- Kong version 1.1.2-apline (docker)

**Comment by bungle** [comment]
@poksiala do you suggest us to add `name` property to `certificate` entity?

**Comment by poksiala** [comment]
Yes that would be great
